examples/advanced/cross-validation-without-training/README.md

```diff
@@ -0,0 +1,31 @@
+# Cifar10_fedavg example re-run cross site validation using the previous trained results
+
+## Introduction
+
+This example shows how to re-run the NVFlare cross-site validation without the training workflow, making use of the previous run results. The example uses the cifar10_fedavg NVFlare job configuration.
+
+### How is the example built
+
+This example uses the exact same NVFlare job definition of cifar10_fedavg, which contains the training workflow, followed by the cross-site validation workflow. These are the steps of changes made to enable re-run the cross-site validation without the training.
+
+1. removed the scatter_gather_ctl workflow from the config_fed_server.json
+
+2.  In the PTFileModelLocator, add the following lines to initialize and create the persistence_manager instance in the model_persister. This is a NVFlare core codes change.
+
+        self.model_persistor.load_model(fl_ctx)
+3. Change the config_fed_server.json to add the "global_model_file_name" and "best_global_model_file_name" with the absolute paths to the global model and best global model locations. Also add  "model_dir" to the configuration. 
+```
+                "global_model_file_name": "{MODEL_DIR}/FL_global_model.pt",
+                "best_global_model_file_name": "{MODEL_DIR}/best_FL_global_model.pt"
+```
+
+4. In order to allow the client to locate the local model and local best model for cross-validation, modify the CIFAR10ModelLearner to add a "model_dir" optional argument. When this "model_dir" is provided, CIFAR10ModelLearner will locate the local models in this folder.
+
+5. Change the config_fed_client.json to include the "model_dir" configuration.
+
+
+## Instructions
+
+The previous run server global models and client local models are stored in the "models" folder. 
+
+run the ./setup.sh to set up the proper config_fed_client.json and proper config_fed_server.json. Then you can run this job for cross-site validation without training workflow.
```

Comment:
Instead of using a "setup.sh" script, I think we could use our "job create cli" to do the replacement?

Reply:
that could be an alternative way to set up. They are not exclusive. We can add that later.

Reply:
No need for the setup process any more. Just define system variables for the model location.